Tasks/VsTestV3/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -90,6 +90,8 @@
   "loc.input.label.rerunMaxAttempts": "Maximum # of attempts",
   "loc.input.help.rerunMaxAttempts": "Specify the maximum # of times a failed test should be retried. If a test passes before the maximum # of attempts is reached, it will not be rerun further.",
   "loc.messages.VstestLocationDoesNotExist": "The location of 'vstest.console.exe' specified '%s' does not exist.",
+  "loc.input.label.donotPublishTestResults": "Disable publishing test results",
```

Comment:
I think these should be added to other locals as well

Reply:
@virekumar, other localizations will be automatically updated by the localization pipelines soon after the en-US changes will be merged.